specs/MultiProfile.md

```diff
@@ -509,13 +662,26 @@ namespace Microsoft.Web.WebView2.Core
 
         String ProfilePath { get; };
 
+        
         CoreWebView2CookieManager CookieManager { get; };
         
         [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Profile7")]
         {
             void Delete();
             event Windows.Foundation.TypedEventHandler<CoreWebView2Profile, Object> Deleted;
         }
+        
+        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Profile2")]
+        {
+            CoreWebView2CookieManager CookieManager { get; };
```

Comment:
The diff appears to be adding the CookieManager property that already exists.  (maybe a merge or view issue?)

Reply:
Yes, it's a merge issue, resolved it, thanks.